src/components/common/HelpSection.vue

```diff
@@ -1,32 +1,26 @@
 <template>
-  <div v-if="helpSection">
-    <span class="help-btn" @click="helpToggle = !helpToggle">
-      <v-icon color="primary" class="pr-1">mdi-help-circle-outline</v-icon>
-      <span v-if="!helpToggle">Help with {{ helpTitle }}</span>
-      <span v-else>Hide Help</span>
-    </span>
+  <div class="help-section-component">
+    <div class="help-btn" @click="helpToggle = !helpToggle">
+      <v-icon color="primary">mdi-help-circle-outline</v-icon>
+      <span v-if="!helpToggle" class="pl-2">{{ header }}</span>
+      <span v-else class="pl-2">Hide Help</span>
+    </div>
 
-    <section v-show="helpToggle" class="help-section">
-      <header class="help-header">
-        <h2>Help with {{ helpTitle }}</h2>
-      </header>
+    <v-expand-transition>
+      <section v-if="helpToggle" class="help-section">
+        <header class="help-header">
+          <h2 class="py-4 px-0">{{ header }}</h2>
+        </header>
 
-      <ul class="px-0">
-        <li
-          v-for="(item, index) in helpSection.helpText"
-          class="py-3"
-          :key="index"
-        >
-          {{item}}
-        </li>
-      </ul>
-      <u class="help-btn" @click="helpToggle = !helpToggle"><small>Hide Help</small></u>
-    </section>
+        <p v-for="(item, index) in items" :key="index" class="my-4" v-html="item" />
+
+        <div class="help-btn bottom" @click="helpToggle = !helpToggle">Hide Help</div>
+      </section>
+    </v-expand-transition>
```

Comment:
I started generalizing this to use in other components, but those other components had too many special cases. It's still cleaner than before though :)